lib/rspec/core/project_initializer/spec/spec_helper.rb

```diff
@@ -18,14 +18,6 @@
   # assertion/expectation library such as wrong or minitest
   # assertions if you prefer.
   config.expect_with :rspec do |expectations|
```

Comment:
This looks empty now, maybe put this here?
```ruby
  expectations.max_formatted_output_length = 200
```
It proved to be quite useful.

Reply:
👍  with a comment saying the 200 is the default